src/test/unit/components/validator.test.ts

```diff
@@ -233,9 +235,12 @@ describe('Validation', () => {
     it.each([
       { mockRef: '', expected: 'invalid' },
       { mockRef: null, expected: 'invalid' },
-      { mockRef: 'a', expected: 'invalid' },
-      { mockRef: '%', expected: 'invalid' },
-      { mockRef: '25a', expected: 'invalid' },
+      { mockRef: 'a', expected: 'notANumber' },
+      { mockRef: '%', expected: 'notANumber' },
+      { mockRef: '25a', expected: 'notANumber' },
+      { mockRef: 'a', expected: 'notANumber' },
+      { mockRef: '%', expected: 'notANumber' },
+      { mockRef: '25a', expected: 'notANumber' },
```

Comment:
either you intended to test different values here or accidentally duplicated, please fix

Reply:
Removed duplicates